.github/workflows/dotnet-ci.yml

```diff
@@ -66,7 +68,7 @@ jobs:
         matrix:
           os: [windows-latest]
           configuration: [Release, Debug]
-          dotnet-version: ['8.0.x']
+          dotnet-version: ['8.0.x', '9.0.x']
```

Comment:
Might worth also explore adding the 9.0 port for the current projects.

`<frameworks>net8.0;net9.0;netstandard2.0</frameworks>`.